updates/next.json

```diff
@@ -30,10 +30,18 @@
     },
     {
       "version": "34.20210427.1.0",
+      "metadata": {
+        "rollout": {
+          "start_percentage": 1.0
+        }
+      }
+    },
```

Comment:
Hmm, any reason we're retaining the last two rollouts? I don't think it hurts, but usually we just edit the `version` field of the previous completed rollout.

Reply:
Bahh, this is from the outdated diff; ignore.